### DISCUSSION TITLE
What should the map marker color be (fill and stroke)?

### DISCUSSION THREAD

**Discussion by jonathan-rogers-dev** [NCVirutal-Coding-Club/CollegeAtlas#42] [Q&A] [answered]
## Here is my idea.
![Screenshot_20210313-105706](https://user-images.githubusercontent.com/39242954/111036046-3f733480-83eb-11eb-99d9-87ad729eaba2.jpg)

**Comment by ITCyb** [comment, marked as answer]
I agree, stick with the same purple theme.

**Reply by jonathan-rogers-dev** [reply]
> 
> 
> I agree, stick with the same purple theme.

I made a new commit in development (#41) with this change.
